reporter/src/main/kotlin/reporters/ExcelReporter.kt

```diff
@@ -323,9 +448,10 @@ private fun XSSFSheet.finalize(headerRows: Int, totalRows: Int, totalColumns: In
 
 private const val ELLIPSIS = "[...]"
 
-private fun Collection<*>.joinWithLimit(
+private fun <T> Iterable<T>.joinWithLimit(
         separator: String = " \n",
-        maxLength: Int = 32767 /* This is the maximum Excel cell content length. */
-) = joinToString(separator).let { cellContent ->
+        maxLength: Int = 32767, /* This is the maximum Excel cell content length. */
+        transform: ((T) -> CharSequence)? = null
```

Comment:
Nit: This new parameter seems to be never used. Will it be in future commits?

Reply:
No, removed.